packages/framework/dds-interceptions/src/test/sharedDirectoryWithInterception.spec.ts

```diff
@@ -425,9 +435,9 @@ describe("Shared Directory with Interception", () => {
 			let asserted: boolean = false;
 			try {
 				sharedDirectoryWithInterception.set("color", "green");
-			} catch (error: any) {
+			} catch (error: unknown) {
 				assert.strictEqual(
-					error.message,
+					(error as Error).message,
```

Comment:
@Josmithr 

Is there a better way to typecasting error of type any?

Reply:
This is fine. This is how I've been tackling these.